SonarQube.Bootstrapper/Program.cs

```diff
@@ -58,8 +58,12 @@ private static int PreProcess(IBuildAgentUpdater updater, IBootstrapperSettings
         {
             string downloadBinPath = settings.DownloadDirectory;
 
-            Utilities.EnsureEmptyDirectory(settings.TempDirectory, logger);
-            Utilities.EnsureEmptyDirectory(downloadBinPath, logger);
+            if (!Utilities.TryEnsureEmptyDirectories(logger,
+                settings.TempDirectory,
+                downloadBinPath))
+            {
+                return ErrorCode;
+            }
```

Comment:
We weren't previously catching errors when creating the empty directories so the user would end up with a stack trace. If they were running on a dev box they'd first get a prompt to debug the app.